apps/js_urls/js/build_url.ts

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash';
```

Comment:
More modern practice: import `zip` or `flatMap` directly instead of `_`, to minimize the JS footprint.

Reply:
This example file is now removed. I couldn't find a good place to put in.